build_image.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+oc create -f deploy/crds/neutron_v1_neutronovsagent_crd.yaml
+oc create -f deploy/crds/neutron_v1_neutronsriovagent_crd.yaml
+oc create -f deploy/crds/neutron.openstack.org_ovsnodeosps_crd.yaml
+oc create -f deploy/crds/neutron.openstack.org_ovncontrollers_crd.yaml
+make
+operator-sdk build --image-builder buildah "${BUILD_IMAGE:-quay.io/openstack-k8s-operators/neutron-operator:latest}"
+sed -i 's|REPLACE_IMAGE|"${BUILD_IMAGE:-quay.io/openstack-k8s-operators/neutron-operator:latest}"g' deploy/operator.yaml
+podman push --authfile "${REGISTRY_AUTH}" "${BUILD_IMAGE:-quay.io/openstack-k8s-operators/neutron-operator:latest}"
```

Comment:
ah sorry, I think my comment was not clear. what I meant is we build a specific version (maybe this should be a required input) and push the version, but also tag the new version as latest with an additional tag. so that we could just use the latest in our dev automation

Reply:
like in https://docs.quay.io/guides/tag-operations.html